code/obj/artifacts/artifact_items/power_cell.dm

```diff
@@ -20,7 +24,34 @@
 			var/datum/artifact/A = src.artifact
 			src.maxcharge = rand(15,1000)
 			src.maxcharge *= 100
+			src.chargeCap = src.maxcharge
 			A.react_elec[2] = src.maxcharge
+
+			// effects
+			src.effectProbModifier = 1/rand(10,50) 	// probability
+			switch(A.artitype.name)										// noise
+				if ("martian")
+					src.noise = pick('sound/voice/babynoise.ogg', 'sound/voice/animal/bugchitter.ogg', 'sound/voice/blob/blobdeath.ogg', 'sound/voice/farts/frogfart.ogg', 'sound/effects/splort.ogg')
+				if ("ancient")
+					src.noise = pick('sound/effects/electric_shock_short.ogg', 'sound/effects/creaking_metal2.ogg','sound/machines/weapons-reloading.ogg', 'sound/machines/glitch1.ogg','sound/machines/glitch2.ogg', 'sound/machines/glitch3.ogg','sound/machines/glitch4.ogg', 'sound/machines/glitch5.ogg', 'sound/machines/scan.ogg')
+				if ("wizard")
+					src.noise = pick('sound/weapons/airzooka.ogg', 'sound/misc/newsting.ogg', 'sound/misc/chair/glass/scoot5.ogg', 'sound/misc/chair/glass/scoot2.ogg')
+				if ("precursor") // what does precursor stuff even sound like???
+					src.noise = pick('sound/effects/singsuck.ogg', 'sound/effects/screech_tone.ogg')
+
+			if(prob(maxcharge/1000)) 									// the more charge the bigger the chance it does dumb stuff
+				switch(A.artitype.name) 								// leakage
+					if ("martian")
+						src.leakChem = pick("space_fungus","blood","vomit","gvomit","urine","meat_slurry","grease","butter","synthflesh","bread","poo","ants","spiders")
+					if ("ancient")
+						src.leakChem = pick("voltagen","ash","cleaner", "oil", "thermite", "acid", "fuel", "nanites", "radium", "mercury")
```

Comment:
Artcells are mostly used by borgs. I think having them leak dangerous chems when in use goes against standard AI laws, and probably isn't a good idea. Maybe remove voltagen, thermite, acid, fuel, nanites, radium, and mercury, and add some other thematic but harmless chems?

Reply:
My intent was for non-rogue borgs to not use those. Since you usually get a few artcells at once I assumed they'd just use one that didn't have harmful side effects.
And you can still put them into stuff like apcs.

Reply:
> My intent was for non-rogue borgs to not use those. Since you usually get a few artcells at once I assumed they'd just use one that didn't have harmful side effects.
>And you can still put them into stuff like apcs.

Borgs (and there usually are multiple) won't know whether the side effects are harmful until they use them, and at that point it's a bit of a sunken cost fallacy, "I've already done this, and this battery is so much better than a regular one, so...". This applies to both the dangerous chems and the arcflashes, which I'm pretty sure can heat up and explode stuff like welding fuel. I don't think APCs should leak dangerous chems either.

Reply:
You can scan the cells before using them with a reagent scanner, it will tell you what they contain.
And, tbh, I kind of added the sparks because I found the idea of something igniting a puddle funny, artifacts are supposed to sometimes be dangerous, y'know?

But if you think it's bad, I guess I'll change it later. v_v

Reply:
I think that being able to scan it solves all problems of accidentially messing up the station by gifting one to robotics.

Research can scan for dangerous stuff inside and either keep it for themselves or a traitor might want to use this for evil acts.

Love it. Sparks are nice.

Reply:
Do the effects fire while inside a cyborg charging station? If not, worst case is that a cyborg has to scream for someone to take it out of them.

Reply:
They would trigger whole inside the station, yeah. But not sure if the chems would leak out of the station. 🤔 
Or if the sparks would set off the welding fuel inside the station.
🤷‍♀️

Reply:
The fact that dangerous reagents are mostly tied to just a single artifact type and the fact that you can reagent scan them makes this fine imo.

Reply:
Also imo maybe the cell could have a small chance (chosen in New) to leak it as a smoke cloud instead.

Reply:
I am open to the possibility of adding rare smoke cells, I just didn't want to make things too similar to chemical bombs, but I guess it'd be fine.

Also, flourish, why do you not want apcs to leak fluids? I found it really cool in testing.
I mean, energy cells are basically the fun-sized version of generator artifacts, and those break windows, shake everything, set things on fire, and instantly vaporize people that are not wearing gloves.
As such I found my implementation pretty tame, and just assumed cyborgs would forgo any dangerous cells.

Reply:
APCs are fine. You can scan them, fair enough, I'd still like the sparks removed though, at least when in borgs.

Reply:
I think the sparks being rarer and in non-borgs only would be good, yea.